src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -182,15 +182,20 @@ public void deleteNetworkModifications(UUID groupUuid, List<UUID> modificationsU
     }
 
     @Transactional
-    public void moveModifications(UUID groupUuid, UUID originGroupUuid, UUID before, NetworkInfos networkInfos, ReportInfos reportInfos, List<UUID> modificationsToMove, boolean canBuildNode) {
+    public UpdateModificationGroupResult moveModifications(UUID groupUuid, UUID originGroupUuid, UUID before, NetworkInfos networkInfos, ReportInfos reportInfos, List<UUID> modificationsToMove, boolean canBuildNode) {
         List<ModificationInfos> movedModifications = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, before)
-            .stream()
-            .map(ModificationEntity::toModificationInfos)
-            .collect(Collectors.toList());
-        if (canBuildNode && !movedModifications.isEmpty() && networkInfos.isVariantPresent()) { // TODO remove canBuildNode and return NetworkDamages() ?
+                .stream()
```

Comment:
Why change the indetation of these 3 lines ?

Reply:
It's the Intellij formatter, I didn't notice, I set back the previous indentation